### PR TITLE
Add FHIR repository connection

### PR DESCRIPTION
**Pull request by Thiyan**
# Purpose
> $subject

### CODE REVIEW

**Comment by sameeragunarathne** [comment]
Need to have vscode based screenshots and also try out steps should reflect to use vscode

File: en/docs/fhir/guides/fhir-repository-connector.md [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+
+``` curl -v -X GET http://localhost:8290/fhir/r4 ```
+
+## Error Handling in the FHIR Repository Connector
+
+If there is an error occurring in the connector, it will throw a synapse exception, and it will trigger the fault sequence which is inside the integration project.
+
+### Fault sequence
+
+The integration studio project contains a fault sequence to create the operation outcome and serialize the operation.
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/fhir-repository-connector.md]: micro integrator vs code plugin

Comment (Thiyan) [reply en/docs/fhir/guides/fhir-repository-connector.md]: Added

File: en/docs/fhir/guides/fhir-repository-connector.md [hunk 2]
```diff
@@ -0,0 +1,1082 @@
+# Introduction
+
+Fast Healthcare Interoperability Resources (FHIR) is an interoperability standard for electronic exchange of healthcare information. The WSO2 FHIR Repository connector can be used to seamlessly integrate with a FHIR repository of your choice.
+
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/fhir-repository-connector.md:4]: Need to have note on two implementation flavours

File: en/docs/fhir/guides/fhir-repository-connector.md [hunk 3]
```diff
@@ -0,0 +1,1082 @@
...
+=== "Micro Integrator"
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/fhir-repository-connector.md:7]: Need to add tab for ballerina as well and how to use bal FHIR connector for the same use case

Comment (Thiyan) [reply en/docs/fhir/guides/fhir-repository-connector.md:7]: It's been already added

File: en/docs/fhir/guides/fhir-repository-connector.md (hunk 2 above)
Comment (Thiyan) [reply en/docs/fhir/guides/fhir-repository-connector.md:4]: Added

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**